onebusaway-api-webapp/src/main/java/org/onebusaway/api/impl/CustomJsonLibHandler.java

```diff
@@ -92,8 +91,9 @@ public String getContentType() {
                 }
                 // we used to set charset for callbacks
                 // after Jackson upgrade to 2.12.0 this no longer works
-                //return "application/json;charset=" + this.defaultEncoding;
-                return "application/json";
+                return "application/json; charset=" + this.defaultEncoding;
```

Comment:
I'm confused by this change given that the comment immediately above it is contradictory.